app/Exceptions/Handler.php

```diff
@@ -43,13 +43,13 @@ public function report(Exception $exception)
      * Render an exception into an HTTP response.
      *
      * @param  \Illuminate\Http\Request  $request
-     * @param  \Exception  $exception
+     * @param  \Throwable  $exception
      * @return \Symfony\Component\HttpFoundation\Response
      *
-     * @throws \Exception
+     * @throws \Throwable
      */
-    public function render($request, Exception $exception)
+    public function render($request, Throwable $exception)
     {
         return parent::render($request, $exception);
     }
-}
+}
```

Comment:
They deprecated the class `Exception` and replaced it with a a more robust `Throwable` in Laravel 7.x. This was apparently the only breaking change that occurred that needed an update.